static/src/javascripts/lib/report-error.spec.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import reportError from './report-error';
+
+describe('report-error', () => {
+    const error = new Error('Something broke.');
+    const meta = {
+        test: true,
+    };
+
+    test('Does not throw an error', () => {
+        expect(() => {
+            reportError(error, meta, false);
+        }).not.toThrowError(error);
+    });
+
+    test('Throws an error', () => {
+        expect(() => {
+            reportError(error, meta);
+        }).toThrowError(error);
+    });
```

Comment:
[Mock](http://facebook.github.io/jest/docs/jest-object.html#jestmockmodulename-factory-options) Raven, [spy](http://facebook.github.io/jest/docs/mock-function-api.html) on `captureException`:
```js
jest.mock('lib/fetch', () => {
  captureException: jest.fn()
});
```
Import the mocked version of Raven in the spec:
```js
import fakeRaven from 'lib/raven';
```
[Test the spy is called](http://facebook.github.io/jest/docs/expect.html#tohavebeencalledwitharg1-arg2-):
```js
expect(fakeRave.captureException).toHaveBeenCalledWith(error, meta);
```